discovery_engine_core/web-api/src/handlers.rs

```diff
@@ -50,7 +50,7 @@ pub(crate) async fn handle_user_interaction(
 ) -> Result<impl warp::Reply, Rejection> {
     if let Some(document) = db.documents_by_id.get(&body.document_id) {
         db.user_state
-            .update_positive_cois(&user_id, |positive_cois| {
+            .update_positive_cois(&body.document_id, &user_id, |positive_cois| {
```

Comment:
🎨  you could also use `document.id`

Reply:
i rebased this onto #616 and applied the change, before this was still a uuid